src/v3/types.ts

```diff
@@ -154,3 +162,20 @@ export interface V3MockServer {
   url: string;
   id: string;
 }
+
+export enum HTTPResponseStatusClass {
+  // Informational responses (100–199)
+  Information = 'information',
+  // Successful responses (200–299)
+  Success = 'success',
+  // Redirects (300–399)
+  Redirect = 'redirect',
+  // Client errors (400–499)
+  ClientError = 'clientError',
+  // Server errors (500–599)
+  ServerError = 'serverError',
+  // Non-error response(< 400)
+  NonError = 'nonError',
+  // Any error response (>= 400)
+  Error = 'error',
```

Comment:
I think these would be better named `2XX`, `4XX`, etc.

Reply:
I had considered that also, I'm glad you mentioned it.

I thought that might be too casual, so I stuck with the definitions in the rust core. Presumably, the first 5 at least were named after the actual classes (e.g. https://developer.mozilla.org/en-US/docs/Web/HTTP/Status).